anyway/infographics_utils.py

```diff
@@ -338,9 +338,11 @@ def localize_after_cache(request_params: RequestParams, items_list: List[Dict])
     res = []
     for items in items_list:
         if "name" in items:
-            res.append(
-                get_widget_class_by_name(items["name"]).localize_items(request_params, items)
-            )
+            widget_class = get_widget_class_by_name(items["name"])
+            if widget_class:
+                res.append(
+                    widget_class.localize_items(request_params, items)
+                )
```

Comment:
This will avoid the error, though that widget will not be included in the result.
I think it happens when we change the name of the widget. I think we did it in a single widget for street, and now we do not do it.